topics/single-cell/tutorials/scatac-batch-correction-snapatac2/tutorial.md

```diff
@@ -0,0 +1,530 @@
+>        - *"Color"*: `batch`
+>        - *"Height of the plot"*: `500`
+>
+> 4. {% icon galaxy-pencil %} Rename the generated image to `spectral-UMAP-No Batch correction`
+>
+> 5. {% icon galaxy-eye %} Inspect the `.png` output
+>
+>  ![UMAP plot before batch correction]({% link topics/single-cell/images/scatac-batch-correction-snapatac2/batch-umap-no_harmony-spectral.png %})
+> 
+> > <question-title></question-title>
+> >
+> > 1. How are batch effects visible in this plot?
+> > 2. How would a plot without batch effects look like?
+> >
+> > > <solution-title></solution-title>
+> > >
+> > > 1. Batch effects are visible as parts of the projection where the different samples are not distributed evenly. For example the large mass in the upper right corner of the plot consists of seperate groups for each batch. It is very likely, that the differences between these groups are technical artefacts, due to batch effects and do not represent "real" biological differences between the cells of different samples. 
+> > > 2. A plot without batch effects does not exhibit groups of cells from a single sample. Instead, most parts of the projection would consist of evenly distributed cells from all samples.  
+> > >
+> > {: .solution}
+>  {: .question}
+>
+>
+{: .hands_on}
+
+# Batch correction
+After confirming that batch effects have affected our samples, we should remove them with correction algorithms. **SnapATAC2** offers three algorithms for batch correction: *Harmony*, *MNC-correct* and *Scanorama*. In order to determine the best-suited algorithm for your specific dataset, the batch correction outputs of different algorithms should be compared. 
+
+> <details-title> Batch correction algorithms </details-title>
+> - Batch correction algorithms aim to correct the cell-by-feature count matrix against batch-specific differences between samples. 
+> - Batch effects can arise from many different technical sources: variation in sequencing lanes, plates, protocol, handling. Additionally, biological factors can also be regarded as batch effects, for example tissue types, species and inter-individual variation {% cite Luecken2021 %}. 
+> - Many different correction algorithms have been developed, although most methods for scATAC-seq have been adapted from scRNA-seq batch removal algorithms. 
+> - *Harmony* {% cite Korsunsky2019 %} is a principle component analysis (PCA)-based method which utilizes the previously generated lower-dimensional data, to assign cells into new clusters. The grouping of cells into clusters favors multi-sample clusters, in order to integrate the datasets. Linear correction factors are calculated for each batch and cluster, and the cells are moved to the corrected positions. The preceding steps are iterated, until optimal batch correction is achieved. 
+>
+>   ![Graphical abstract of Harmony batch correction]({% link topics/single-cell/images/harmony-graphical-abstract.png %})
```

Comment:
It looks like the path for this image is missing a subfolder: (same for other links in this box)

```suggestion
>   ![Graphical abstract of Harmony batch correction]({% link topics/single-cell/images/scatac-batch-correction-snapatac2/harmony-graphical-abstract.png %})
```